tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/TestData/review.cmd

```diff
@@ -0,0 +1,21 @@
+@echo off
```

Comment:
Please add some comments into this file describing what it does. I've no idea!

Reply:
Done